mmv1/products/accesscontextmanager/ServicePerimeterDryRunEgressPolicy.yaml

```diff
@@ -0,0 +1,164 @@
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'access_context_manager_service_perimeter_dry_run_egress_policy'
+    skip_test: true
+autogen_async: true
+exclude_tgc: true
+# Skipping the sweeper due to the non-standard base_url and because this is fine-grained under ServicePerimeter
+skip_sweeper: true
+exclude_import: true
+id_format: '{{perimeter}}'
+import_format: ['{{perimeter}}']
+mutex: '{{perimeter}}'
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  custom_import: templates/terraform/custom_import/access_context_manager_service_perimeter_ingress_policy.go.erb
+  pre_update: templates/terraform/pre_create/access_context_manager_dry_run_resource.go.erb
+  pre_create: templates/terraform/pre_create/access_context_manager_dry_run_resource.go.erb
+  pre_delete: templates/terraform/pre_create/access_context_manager_dry_run_resource.go.erb
+parameters:
+  - !ruby/object:Api::Type::ResourceRef
+    name: 'perimeter'
+    resource: 'ServicePerimeter'
+    imports: 'name'
+    description: |
+      The name of the Service Perimeter to add this resource to.
+    required: true
+    url_param_only: true
+properties:
+  - !ruby/object:Api::Type::NestedObject
+    name: 'egressFrom'
+    description: |
+      Defines conditions on the source of a request causing this `EgressPolicy` to apply.
+    properties:
+      - !ruby/object:Api::Type::Enum
+        name: 'identityType'
+        description: |
+          Specifies the type of identities that are allowed access to outside the
+          perimeter. If left unspecified, then members of `identities` field will
+          be allowed access.
+        values:
+          - :ANY_IDENTITY
+          - :ANY_USER_ACCOUNT
+          - :ANY_SERVICE_ACCOUNT
+      - !ruby/object:Api::Type::Array
+        name: 'identities'
+        description: |
+          A list of identities that are allowed access through this `EgressPolicy`.
+          Should be in the format of email address. The email address should
+          represent individual user or service account only.
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::Array
+        name: 'sources'
+        description: 'Sources that this EgressPolicy authorizes access from.'
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: 'accessLevel'
+              description: 'An AccessLevel resource name that allows resources outside the ServicePerimeter to be accessed from the inside.'
+      - !ruby/object:Api::Type::Enum
+        name: 'sourceRestriction'
+        description: 'Whether to enforce traffic restrictions based on `sources` field. If the `sources` field is non-empty, then this field must be set to `SOURCE_RESTRICTION_ENABLED`.'
+        values:
+          - :SOURCE_RESTRICTION_UNSPECIFIED
```

Comment:
the _UNSPECIFIED value is typically not useful for Terraform users to set. This should probably be removed

Reply:
Removed all isntances, thanks!

Reply:
I'm still seeing this one